crates/aptos/src/test/mod.rs

```diff
@@ -334,7 +334,13 @@ impl CliTestFramework {
 
     pub async fn add_stake(&self, index: usize, amount: u64) -> CliTypedResult<TransactionSummary> {
         AddStake {
-            txn_options: self.transaction_options(index, None),
+            txn_options: self.transaction_options(
```

Comment:
@gregnazario I managed to fix all smoke_tests using ugly fixes like this. Please let me know if you are okay with them and feel free to speak out if I'm being excessive here.

I still don't understand why gas estimation doesn't work in these cases. Maybe it's something worth looking into

Reply:
Talked with Greg offline. Our plan is to go ahead with the hacks in this PR and Greg will take care of them after he fixes gas estimation.